ui/src/components/ConfirmDeleteTask.vue

```diff
@@ -0,0 +1,38 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+<template>
+  <NsModal
```

Comment:
Let's use `NsDangerDeleteModal` when deleting something

Reply:
there is no data deletion ?????